PANDORA/Database/Generate_reverse_templates.py

```diff
@@ -0,0 +1,526 @@
+import numpy as np
+from simtk.openmm.app import PDBFile, Modeller, ForceField, Simulation
+from pdbfixer import PDBFixer
+from simtk import openmm
+from openmm import LangevinIntegrator, Platform
+from openmm.unit import kelvin, picosecond, femtoseconds
+from Bio.PDB import PDBParser, PDBIO
+import os
+from typing import List, Dict, Tuple
+
```

Comment:
As this is a stand-alone script that users are not necessarily supposed to use, it should maybe list in here or in a separate README the required packaged, package versions and so on, for the packages that are different from the ones normally installed by PANDORA.

Reply:
will do, thx!